integration-tests/locales/some/src/test/java/io/quarkus/locales/it/LocalesIT.java

```diff
@@ -94,6 +98,21 @@ public void testDefaultLocale() {
                 .log().all();
     }
 
+    @Test
+    @DisableIfBuiltWithGraalVMOlderThan(value = GraalVMVersion.GRAALVM_24_2_0)
+    public void testDefaultLocaleAfter24_1() {
+        RestAssured.given().when()
+                .get("/default/de-CH")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                /*
+                 * "Schweiz" is the correct name for Switzerland in German.
+                 * German is the default language as per the application.properties.
```

Comment:
```suggestion
                 * "Schweiz" is the correct name for Switzerland in German.
                 * German is the default language as per the `quarkus.test.arg-line` in application.properties.
```

Reply:
+1 for mentioning the directive that is supposed to cause this change in behaviour